Source/WebKit/UIProcess/Automation/mac/WebAutomationSessionMac.mm

```diff
@@ -248,6 +254,7 @@ static WebMouseEventButton automationMouseButtonToPlatformMouseButton(MouseButto
         [eventsToBeSent addObject:[NSEvent mouseEventWithType:downEventType location:locationInWindow modifierFlags:modifiers timestamp:timestamp windowNumber:windowNumber context:nil eventNumber:eventNumber clickCount:2 pressure:WebCore::ForceAtClick]];
         [eventsToBeSent addObject:[NSEvent mouseEventWithType:upEventType location:locationInWindow modifierFlags:modifiers timestamp:timestamp windowNumber:windowNumber context:nil eventNumber:eventNumber clickCount:2 pressure:0.0f]];
     }
+    updateClickCount(button, locationInWindow);
```

Comment:
Could you explain the reasoning behind this? The delta stuff above is clear from the ChangeLog, but I don't think this is explained.

Reply:
Oh yeah, I should update the changeling to explain this too. This call updates `m_lastClickPosition`, which we need to compute the event deltas

Reply:
(I'll update the changelog to reflect that)

Reply:
https://github.com/WebKit/WebKit/commit/e2ea2eb9bfcb731d701adfaac4cd48642a4aadc6